src/judoscale/core/config.py

```diff
@@ -25,12 +23,17 @@ def for_report(self):
         }
 
     def _prepare_logging(self):
+        logger = logging.getLogger('judoscale')
+        print(self.log_level)
         log_level = logging.getLevelName(self.log_level.upper())
-        logging.basicConfig(
-            level=log_level,
-            format="%(levelname)s - [Judoscale] %(message)s",
-            force=True,
-        )
+        logger.setLevel(log_level)
```

Comment:
Will this impact the log level of logs emitted outside of our package (such as logging from the Django app), or is it isolated to our package?

Reply:
It is isolated because the logger here looks at the judoscale namespace in line 26 above.